packages/atomic/src/components/atomic-sort-criteria/readme.md

```diff
@@ -13,6 +13,19 @@
 | `criteria` _(required)_ | `criteria` | The sort criterion/criteria the end user can select/toggle between.  The available sort criteria are: - `relevancy` - `date ascending`/`date descending` - `qre` - `field ascending`/`field descending`, where you must replace `field` with the name of a sortable field in your index (e.g., `criteria="size ascending"`).  You can specify multiple sort criteria to be used in the same request by separating them with a comma (e.g., `criteria="size ascending, date ascending"` ). | `string` | `undefined` |
 
 
+## Dependencies
```

Comment:
It would have been nice if there was a way to make it work for the exact story here also ;) 

In this case, `atomic-sort-criteria` has a soft dependency on atomic-sort-dropdown ... Would be better to have it documented like that somehow than just breaking on usage.

Reply:
Once we get a json reference doc, we could add new doc tags likes @dependsOn or similar to add to the dependencyGraph, currently it appears to be automatically build from the JSX